code/datums/supplypacks.dm

```diff
@@ -678,6 +678,18 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	access = access_security
 	group = "Security"
 
+/datum/supply_packs/smartlaser
+	name = "Smart laser gun crate"
+	contains = list(/obj/item/weapon/gun/energy/laser/smart,
+					/obj/item/weapon/gun/energy/laser/smart,
+					/obj/item/weapon/gun/energy/laser/smart,
+					/obj/item/weapon/gun/energy/laser/smart)
+	cost = 60
+	containertype = /obj/structure/closet/crate/secure/weapon
+	containername = "Smart laser gun crate"
```

Comment:
I've told multiple people to do this already, see inflatables.

The thing is, if they're capitalized they'll be proper nouns, so it'll be `That's Smart laser gun crate` vs `That's a smart laser gun crate`.